src/shared/components/DraggableList/DraggableList.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+
+import { Icon } from '@viaa/avo2-components';
+import classNames from 'classnames';
+
+export interface DraggableListProps {
+	elements: JSX.Element[];
+	onListChange: (updatedList: JSX.Element[]) => void;
+}
+
+export interface DraggableListState {
+	currentlyBeingDragged: JSX.Element | null;
+}
+
+export class DraggableList extends Component<DraggableListProps, DraggableListState> {
```

Comment:
Waarom geen functional component?

Reply:
Bij Functional Components is het niet mogelijk om de state te testen met Enzyme.